### DISCUSSION TITLE
12/27

### DISCUSSION THREAD

**Discussion by BeMatthewsong** [Team-Plants/PLANTS#71] [Daily Scrum]
## 김윤후

- 오늘의 컨디션 : 10점 많이 잤다~
- 어제 진행했거나 완료한 작업 : 대시보드 수정페이지 ui
- 오늘 진행할 작업 : 대시보드 생성 기능
- 막혔던 부분 : x
- 도움이 필요한 부분 : x 
- TMI : 꿈 4가지나 꿨다 ~

## 송민혁

- 오늘의 컨디션 : 6점 
- 어제 진행했거나 완료한 작업 : 대시보드 ui
- 오늘 진행할 작업 : 대시보드ui, 칼럼 추가 기능
- 막혔던 부분 : 플렉스박스, 그리드
- 도움이 필요한 부분 : pr
- TMI : 피자 먹기 성공

## 안혜정

- 오늘의 컨디션 :
- 어제 진행했거나 완료한 작업 : 
- 오늘 진행할 작업 : 
- 막혔던 부분 : 
- 도움이 필요한 부분 : 
- TMI : 

## 전수빈

- 오늘의 컨디션 : 7점 일찍 자서 기분이 좋다 ^^^^^
- 어제 진행했거나 완료한 작업 : 로그인, 회원가입 기능
- 오늘 진행할 작업 : 멘토링 반영해야 함
- 막혔던 부분 : api 부분이 아직 부족함...
- 도움이 필요한 부분 : 
- TMI : 아샷추...빽다방...작업이 잘 될듯...!

## 한슬희

- 오늘의 컨디션 : 8점 
- 어제 진행했거나 완료한 작업 : 마이페이지 ui 
- 오늘 진행할 작업 : 버튼 수정 예정 
- 막혔던 부분 : x
- 도움이 필요한 부분 : x
- TMI : 6시 운동할 예정이었는데 실패...! 3대~~~~~~~

**Comment by hanseulhee** [comment]
아샷추 맛있게따 .. 꿀꺽

**Reply by K-Y-Hoo** [reply]
저도 운동갔다오면서 아샷추 사올려구요..~

**Reply by BeMatthewsong** [reply]
수빈님 아샷추 찐사랑 느껴졌어요

**Reply by hanseulhee** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋ
